apps/server/src/infra/sync/strategy/tsp/tsp-fetch.service.spec.ts

```diff
@@ -12,10 +15,7 @@ import {
 	RobjExportSchuelerMigration,
 	RobjExportSchule,
 	TspClientFactory,
-} from '@src/infra/tsp-client';
-import { OauthConfigMissingLoggableException } from '@src/modules/oauth/loggable';
-import { systemFactory } from '@src/modules/system/testing';
-import { AxiosError, AxiosResponse } from 'axios';
+} from '../../../tsp-client';
```

Comment:
Not the same module.
```suggestion
} from '@src/infra/tsp-client';
```